templates/kapeta/block-type-frontend/webpack.development.config.js.hbs

```diff
@@ -51,7 +50,7 @@ const config = {
     output: {
         path: Path.join(__dirname, "dist"),
         filename: "[name].[contenthash].bundle.js",
-        publicPath: "",
+        publicPath: "/dist/",
```

Comment:
I found out in `web-metrics` that without doing this change the app couldn't run in prod mode. I saw this error in the console:

```
No renderPage method on response object. Using fallback render.
````

And in the returned html I saw this error:

```
window.BACKEND_ERROR = {"errno":-2,"code":"ENOENT","syscall":"stat","path":"/Users/harald/code/kapeta/web-metrics/dist/metrics","expose":false,"statusCode":404,"status":404};
```

Changing the publicPath to `"/dist/"` solved it. So the current latest SDK's require this change to work.